handbook/product/design/index.md

```diff
@@ -67,6 +67,34 @@ While design does adds time at the beginning of a project, it improves velocity
 
 Our design process is documented here: **[Design process](design_process.md)**
 
+## Design Team Operational Guidelines
+
+Rules that help us make decisions in crucial moments of the design process.
+
+- **Always design with the advanced user in mind**
+
+Our users are mostly highly skilled developers who are used to complex interfaces. They are more knowledgeable and need less guidance than regular web users. They also have higher expectations and are less forgiving.
```

Comment:
Sourcegraph is for all developers, so want to clean up wording to not alienate anyone :) 

```suggestion
Our users are developers who are accustomed to using complex interfaces. They are more knowledgeable and need less guidance than regular web users. They also have higher expectations and are less forgiving.
```